paddlenlp/transformers/qwen/modeling.py

```diff
@@ -213,10 +264,7 @@ def forward(
             logn_tensor = self.logn_tensor[:, seq_start:seq_end, :, :]
             query = query * logn_tensor.expand(query.shape)
 
-        query = query.transpose([0, 2, 1, 3])
-        key = key.transpose([0, 2, 1, 3])
-        value = value.transpose([0, 2, 1, 3])
-        attn_output, attn_weight = self._attn(query, key, value, attention_mask)
+        attn_output, attn_weight = self._attn(query, key, value, self.config, attention_mask)
```

Comment:
```suggestion
        attn_output, attn_weight = self._attn(query, key, value, attention_mask)
```
self.config, 可以在 _attn 中获取，不需要传

Reply:
Done